app/src/androidTest/java/com/lambdasoup/quickfit/util/DatabasePreparationTestRule.java

```diff
@@ -86,39 +86,39 @@
         s11.put(ScheduleEntry.COL_HOUR, 18);
```

Comment:
Some sort of builder would probably be nice and give a way of expressing the point for each row

Reply:
Great idea! I'd open an issue for that, but with #45 planned for the near future, I see no point there (same as in other possible improvements to the whole db query/result handling, like extension methods on `Cursor` for accessing specific columns).